test/percy-agent-client/dom.test.ts

```diff
@@ -0,0 +1,260 @@
+import { expect } from 'chai'
+import * as cheerio from 'cheerio'
+// @ts-ignore
+import { check, type } from 'interactor.js'
+import * as sinon from 'sinon'
+import DOM from '../../src/percy-agent-client/dom'
+
+// Create valid DOM to pass to the DOM class.
+function createExample(dom: any) {
+  let testContainer = document.querySelector('.test-container')
+
+  if (testContainer) {
+    testContainer.remove()
+  }
+
+  testContainer = document.createElement('div')
+
+  const finalDOM = `
+    <div class="container">
+      <h1>Hello DOM testing</h1>
+      ${dom}
+    </div>
+  `
+
+  testContainer.classList.add('test-container')
+  testContainer.innerHTML = finalDOM
+  document.body.appendChild(testContainer)
+
+  return document
+}
+
+// create a stylesheet in the DOM and add rules using the CSSOM
+function createCSSOM() {
+  const style = document.createElement('style')
+  const testingContainer = document.querySelector('.test-container')
```

Comment:
```suggestion
  const testingContainer = document.querySelector('.test-container') as HTMLElement
```

this resolves the need for `strictNullChecks` to be disabled.